### DISCUSSION TITLE
OMR installed/setup successfully on Virtualbox environment, but cannot access Internet from Main Machine (Windows 10)

### DISCUSSION THREAD

**Discussion by wicckyy** [Ysurac/openmptcprouter#2480] [Q&A] [answered]
Hello,

First of all thank you Yannick and OMR dev team for creating this wonderful piece of software. I followed OneMarcFifty tutorial on YouTube to create VirtualBox network and install OMR. The combined internet runs fine on internal Virtualbox additionally created Lubuntu machine, but I am unable to access combined internet connections from my  external Main Machine (Windows 10).

**Here's my current setup:**

Virtualbox  VM (OMR) running on 192.168.100.1 

**Virtual Box Network Settings:**
_Network Adapter 1: (working)_  
-- Host Only Adapter (192.168.100.45) - DHCP Disabled.

_Adapter 2 (working)_ 
-- Intel Gigabit Lan (bridged from Main Machine Windows 10 --> Connected to ISP Router 1

_Adapter 3  (working)_
-- Wifi Dongle (bridged from Main Machine Windows 10 --> Connected to ISP Router 2 

![OMR](https://user-images.githubusercontent.com/111511550/185401285-2f6d9f85-6dc6-4baa-a09d-28e96cb5720e.png)

Everything is working correctly if I create a virtualbox internal Lubuntu virtual machine with HostOnly adapter. It gets IP address automatically and can browse internet with combined speeds.

### The main machine Windows 10 (on which virtualbox is installed), cannot browse the OMR Router's combined internet, even after specifying static IP address (192.168.100.150) and gateway 192.168.100.1  on LAN connection. The adapter says "Unidentified network" after assigning the static ip/gateway.

On automatic IP configuration, I can access OMR Router page (192.168.100.1) successfully using Chrome on my Windows 10 Main machine. However the Windows uses the ISP router for internet instead of OMR (obviously as no route/gateway to OMR is defined in LAN connection.

@onemarcfifty

**Comment by wicckyy** [comment, marked as answer]
Hello,

I managed to solve the problem.

The configuration was alright.

I just needed to Set the Interface Metric in Lan Adapter from Windows 10.

I changed the metric of "Virtualbox" host adapter to "1" (high priority) and set the Bridged adapters to "2" and "3" respectively.

It basically told Windows 10 to use Virtualbox Host adapter as primary internet connection and use OpenMPTCPRouter bonded connections.

-- I might write a detailed guide in future if anyone wants.

**Reply by lomatik** [reply]
Hello, write please detail guide how to configure VirtualBox for openmptcprouter

**Comment by Ferrar65** [comment]
same problem

**Reply by lomatik** [reply]
did u fix it?
